ocaml/xenopsd/xc/xenops_server_xen.ml

```diff
@@ -433,7 +433,9 @@ module Storage = struct
   (* We need to deal with driver domains here: *)
   let attach_and_activate ~xc ~xs task vm dp sr vdi read_write =
     let vmdomid = vm_of_domid (domid_of_uuid ~xc ~xs (uuid_of_string vm)) in
-    let result = attach_and_activate task vm vmdomid dp sr vdi read_write in
+    let result =
+      attach_and_activate ~task ~_vm:vm ~vmdomid ~dp ~sr ~vdi ~read_write
```

Comment:
It is a bit strange that the parameter is called _vm here, but looks good otherwise.